test/journeys/specs/oneOnOne/messaging.js

```diff
@@ -114,13 +122,94 @@ describe(`Widget Space: One on One`, () => {
 
     it(`sends and deletes message`);
 
-    it(`sends message with pdf attachment`);
+    describe(`File Transfer Tests`, () => {
+      /* eslint no-sync: "off" */
+      const uploadDir = path.join(__dirname, `assets`);
+      const downloadDir = path.join(os.homedir(), `Downloads`);
+      const inputFileElem = `.ciscospark-file-input`;
+      const downloadFileButton = `(//div[@title="Download this file"]/parent::button)[last()]`;
+      const shareButtonElem = `button[aria-label="Share"]`;
+      function sendFileTest(fileName) {
+        clearEventLog(browserRemote);
+        const filePath = path.join(uploadDir, fileName);
+        const fileSize = fs.statSync(filePath).size;
+        const downloadedFile = path.join(downloadDir, fileName);
+        const fileTitle = `//div[text()="${fileName}"]`;
+        if (fs.existsSync(downloadedFile)) {
+          fs.unlinkSync(downloadedFile);
+        }
+        browserLocal.chooseFile(inputFileElem, filePath);
+        browserLocal.click(shareButtonElem);
+        browserRemote.waitForExist(fileTitle, 30000);
+        browserRemote.moveToObject(downloadFileButton);
```

Comment:
The button has to have a mouseover event for it to appear, and become clickable.  MoveToObject accomplishes the mouse over, but click can occur faster than the button becomes visible, resulting in a false negative.  So, I added to waitForVisible to prevent that.